QUEST_JOBSTEP.tsv

```diff
@@ -2544,9 +2544,9 @@ QUEST_JOBSTEP_20150401_002543	Go talk to the Chronomancer Master at the road of
 QUEST_JOBSTEP_20150406_002544	Ah, one more thing.{nl}There is a hidden object at Flude Island of the west forest of Siaulai.{nl}Before you go to Klaipeda, find it first and return it to its original owners.
 QUEST_JOBSTEP_20150406_002545	I can afford it
 QUEST_JOBSTEP_20150406_002546	I don't have money for that
-QUEST_JOBSTEP_20150406_002547	Obtain the book from the two dimensional world at the 1st floor of Tenet Cathedral
-QUEST_JOBSTEP_20150406_002548	The Linker Master told you that if you are competent enough, you would be able to find the two dimensional world at the 1st floor of Tenet Cathedral. Find the book from the two dimensional world and give it to the Linker Master.
-QUEST_JOBSTEP_20150406_002549	Obtain the book from the two dimensional world at the 1st floor of Tenet Cathedral
+QUEST_JOBSTEP_20150406_002547	Obtain the book from the two dimensional world at the 1st floor of Tenet Chapel
```

Comment:
It's singular, two dimensional word refers to the flat world; aka a drawing, painting, book, etc.